Reachability.m

```diff
@@ -152,6 +156,8 @@ -(instancetype)initWithReachabilityRef:(SCNetworkReachabilityRef)ref
         self.reachableOnWWAN = YES;
         self.reachabilityRef = ref;
 
+        CFRetain(self.reachabilityRef);
```

Comment:
Except there's still a potential leak according to the analyzer, since ```SCNetworkReachabilityCreateWithName``` has already increased the retain count.

While I agree that this line is necessary to match the ```CFRelease``` call in dealloc, the ref should also be released in the factory methods to match the create calls.